import_video.py

```diff
@@ -66,8 +63,9 @@ def get_params(self, path):
                     lng = float(match.group('lng'))
                 except: pass
         if not camera:
-            logging.warn('did not detect camera name, assuming "unnamed"')
-            camera = DEFAULT_CAMERA_NAME
+            logging.error('did not detect camera name, assuming "%s"', self.DEFAULT_CAMERA_NAME)
+            sys.exit(1)
+            camera = self.DEFAULT_CAMERA_NAME
```

Comment:
@jhallard I now see that "camera" is the capture group name, which is nice for its simplicity; so let's just have the python code refer to it as camera_name even while retaining the capture group "camera".